test/java/uk/co/real_logic/sbe/codec/java/BitUtilTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2013 Real Logic Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.co.real_logic.sbe.codec.java;
+
+import java.lang.reflect.Field;
+import java.nio.Buffer;
+import java.nio.ByteBuffer;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+public class BitUtilTest 
```

Comment:
I'm not sure.  I wrote a small test for it as it was the smallest component I could initially find. But on closer inspection I'm not sure this class (or this method) is even used anywhere.

I could remove the class and test if that would be preferable?

Cheers!